CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
+
+
 <!-- towncrier release notes start -->
 
+## v7.5.1
+
+*2025-09-26*
+
+### Bug fixes
+
+- Fix static parsing of {file}`pyproject.toml` data when the
+  {file}`pyproject.toml` is supplied as a relative path -- by {user}`sirosen`.
+
+  *PRs and issues:* {issue}`2215`, {issue}`2221`, {issue}`2233`
+
+- The "via" paths in `pip-compile` output for requirements discovered from
+  `pyproject.toml` data are now written in POSIX format -- by {user}`sirosen`.
+
+  *PRs and issues:* {issue}`2221`
+
+- Fixed a bug which removed slashes from URLs in ``-r`` and ``-c`` in the output
+  of ``pip-compile`` -- by {user}`sirosen`.
+
+  *PRs and issues:* {issue}`2223`
+
+- Fix an incompatibility with ``click >= 8.3`` which made ``pip-compile`` display incorrect
+  options in the compile command in output headers -- by {user}`sirosen`.
+
+  *PRs and issues:* {issue}`2235`
+
+### Improved documentation
+
+- ReadTheDocs builds for `pip-tools` no longer include htmlzip and pdf outputs -- by :user:`sirosen`.
+
+  *PRs and issues:* {issue}`2218`
+
+### Contributor-facing changes
+
+- `pip-tools` now tests on and officially supports `pip` version 25.2 -- by {user}`sirosen`.
```

Comment:
The "officially supports" part should probably be also mentioned in the features section.

Reply:
I moved this to the Features section in my last commit, but that felt sort of wrong. I'm going to add a tiny patch commit which splits this into two items.

If you approve of the result, I'll squash things.

Reply:
Ack, marking as resolved.